### PR TITLE
[new] add angular-loading-bar 0.6.0

### PR DESCRIPTION
**Pull request by int64ago**
Ref: https://github.com/chieffancypants/angular-loading-bar

### CODE REVIEW

**Comment by PeterDaveHello** [comment]
You should add filename field for it and minify the files, thanks.

**Comment by int64ago** [comment]
Sorry, what does "filename field" mean?

**Comment by drewfreyling** [comment]
@int64ago the package.json should contain a `filename` field as this is used as the default file for the library. This is why the tests would be failing.

**Comment by int64ago** [comment]
@drewfreyling It tests ok now, thx :)

File: ajax/libs/angular-loading-bar/package.json [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+  "author": "Wes Cruver",
+  "license": "MIT",
+  "bugs": {
+    "url": "https://github.com/chieffancypants/angular-loading-bar/issues"
+  },
+  "homepage": "https://chieffancypants.github.io/angular-loading-bar",
+  "npmName": "angular-loading-bar",
+  "npmFileMap": [
+    {
+      "basePath": "/src/",
```
Comment (PeterDaveHello) [inline ajax/libs/angular-loading-bar/package.json]: Should be `build`

**Comment by PeterDaveHello** [comment]
Looks not okay, you commit the minified file but refer to unminified file in auto-update.

**Comment by int64ago** [comment]
@PeterDaveHello 
> If the library doesn't already provide a minified version, cdnjs's preferred JavaScript minifier is UglifyJS
What should I do?
And what does "Should be build" mean?
Thx

**Comment by PeterDaveHello** [comment]
I meant change basePath to `build`.

**Comment by int64ago** [comment]
Would you please add angular-loading-bar personally, ignore my pull request, thx a lot !

**Pull request closed without merging**